stock_release_channel_process_end_time/models/stock_picking.py

```diff
@@ -8,28 +10,44 @@ class StockPicking(models.Model):
 
     _inherit = "stock.picking"
 
-    schedule_date_prior_to_channel_process_end_date_search = fields.Boolean(
+    # TODO: move this field to stock_release_channel so that we don't have to alter
+    # _field_picking_domains.
+    scheduled_date_prior_to_channel_end_date_search = fields.Boolean(
         store=False,
-        search="_search_schedule_date_prior_to_channel_process_end_date",
+        search="_search_scheduled_date_prior_to_channel_end_date",
         help="Technical field to search on not processed pickings where the scheduled "
-        "date is prior to the process end date of available channels.",
+        "date is prior to the end date of related channel.",
     )
 
     @api.model
     def fields_get(self, allfields=None, attributes=None):
-        """Hide schedule_date_prior_to_channel_process_end_date_search from
+        """Hide scheduled_date_prior_to_channel_end_date_search from
```

Comment:
Use a comment instead of a docstring.
```suggestion
        """Hide scheduled_date_prior_to_channel_end_date_search from
```